.env.development

```diff
@@ -0,0 +1,6 @@
+DB_NAME=badging_dev
+DB_USER=root
+DB_PASSWORD=rootpwd
+DB_HOST=127.0.0.1
+DB_DIALECT=mysql
+PORT=8000
```

Comment:
Here we are still missing the value for `AUGUR_API_KEY` but I'm not really sure what that's used for and what value it should have 🤔